src/gurt/dlog.c

```diff
@@ -674,7 +674,7 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 			setlinebuf(stderr);
 			mst.logfd = fileno(stderr);
 		} else {
-			mst.logfd = open(mst.logfile, log_flags, 0666);
+			mst.logfd = open(mst.logfile, log_flags, 0644);
```

Comment:
Why do you keep ug+r ? Also, what is the JIRA ticket associated with this PR ?

Reply:
It's a client log file, group user and others are able to read it given that only the owner can write it and no confidential data in the log. We had some discussion last couple of days in ESAD build/test forum.

https://jira.hpdd.intel.com/browse/DAOSSDE-58

Reply:
hmmm, the ticket link is still unreachable for me (infamous "404 - Not Found" page) :-(
I remember that we have been talking about the different way this issue should be addressed to get PR reviewers also get access to associated JIRA, but is there any process in place now ? Again, I remember that it could be possible to define groups (users, developers,...) of people who can get access, like having all DAOS teammates to be put in developer group so they could be the only to get access to some tickets...